code/src/java/pcgen/core/display/DisplayUtilities.java

```diff
@@ -36,41 +39,20 @@ private DisplayUtilities()
 	 * The items will be joined in the order determined by the ordering of the
 	 * given Collection.
 	 *
-	 * This method is value-semantic. DisplayUtilities will not maintain a
-	 * reference to or modify the given Collection.
-	 *
 	 * @param cdoCollection
 	 *            An Collection of CDOMObjects
-	 * @param separator
-	 *            The separating string
 	 * @return A 'separator' separated String containing the Display Name of the
 	 *         given Collection of CDOMObject objects
 	 */
 	@Deprecated
+	@Contract(pure = true)
 	public static String joinDisplayName(
-		Collection<? extends CDOMObject> cdoCollection, String separator)
+		Collection<? extends CDOMObject> cdoCollection)
```

Comment:
This is one of those places where I think in-lining is questionable and requires some thought in the process.... under what circumstances do you not in-line?